### PR TITLE
enable sending registration email

### PR DESCRIPTION
**Pull request by SimonLab**
see #103

### CODE REVIEW

**Review by Danwhy — APPROVED** [review]
Good :+1: 
We should probably implement some kind of error handling at some point in case the email doesn't work

File: config/config.exs [hunk 1]
```diff
@@ -14,7 +14,8 @@ config :research_resource,
   qualtrics_mailinglist_id: System.get_env("QUALTRICS_MAILINGLIST_ID"),
   qualtrics_survey_id: System.get_env("QUALTRICS_SURVEY_ID"),
   qualtrics_distribution_id: System.get_env("QUALTRICS_DISTRIBUTION_ID"),
-  id_prefix: "TTRR"
+  id_prefix: "TTRR",
+  email: "tt.research.resource@gmail.com"
```
Comment (samhstn) [inline config/config.exs:18]: Really good to do this 👍 @SimonLab, I'm taking out these variables in my current project refactor

File: web/controllers/user_controller.ex [hunk 2]
```diff
@@ -13,10 +13,7 @@ defmodule ResearchResource.UserController do
     changeset = User.registration_changeset(%User{}, user_params)
     case Repo.insert(changeset) do
       {:ok, user} ->
-        subject = "Welcome to Research Resource"
-        message = "Welcome to Research Resource, #{user.first_name}"
-        # mail = ResearchResource.Email.send_email(user.email, subject, message)
-        # |> ResearchResource.Mailer.deliver_now()
+        send_registration_email(user)
```
Comment (samhstn) [inline web/controllers/user_controller.ex:16]: Nice 👍

**Comment by samhstn** [comment]
Looks good @SimonLab

**Pull request merged**